core/base/src/TApplication.cxx

```diff
@@ -1052,11 +1052,23 @@ void TApplication::Help(const char *line)
       gInterpreter->ProcessLine(line);
       Printf("\nROOT special commands.");
       Printf("==============================================================================");
-      Printf("   .pwd                : show current directory, pad and style");
-      Printf("   .ls                 : list contents of current directory");
-      Printf("   .which [file]       : shows path of macro file");
+      Printf("   .credits            : show credits");
+      Printf("   .demo               : launch GUI demo");
       Printf("   .help Class         : opens the reference guide for that class (or .?)");
       Printf("   .help Class::Member : opens the reference guide for function/member (or .?)");
+      Printf("   .help edit          : show line editing shortcuts (or .?)");
+      Printf("   .license            : show license");
+      Printf("   .ls                 : list contents of current TDirectory");
+      Printf("   .pwd                : show current TDirectory, pad and style");
+      Printf("   .quit (or .exit)    : quit ROOT (long form of .q)");
+      Printf("   .R [user@]host[:dir] [-l user] [-d dbg] [script]\n"
+             "                       : launch process in a remote host");
+      Printf("   .qqq                : quit ROOT - mandatory");
+      Printf("   .qqqqq              : exit process immediately");
+      Printf("   .qqqqqqq            : abort process");
+      Printf("   .which [file]       : shows path of macro file");
+      Printf("   .![OS_command]      : executes OS-specific shell command");
+      Printf("   .!root -?           : print ROOT usage (CLI options)");
```

Comment:
Yes, it's a side effect. It would be more coherent to have a `.usage` command, but for simplicity, I thought to just make it visible reusing already implemented functionality.